src/types/io.ts

```diff
@@ -668,6 +668,7 @@ export interface AoARIOWrite extends AoARIORead {
     params: AoRedelegateStakeParams,
     options?: WriteOptions,
   ): Promise<AoMessageResult>;
+  writeAction(options?: WriteOptions): Promise<AoMessageResult>;
```

Comment:
this offers some utility here, but we could instead just make an  `AoProcess` in CLI layer and call `send`

Reply:
17136e4